boa_cli/src/debug/optimizer.rs

```diff
@@ -44,24 +44,28 @@ fn set_statistics(_: &JsValue, args: &[JsValue], context: &mut Context<'_>) -> J
 }
 
 pub(super) fn create_object(context: &mut Context<'_>) -> JsObject {
-    let get_constant_folding =
-        FunctionObjectBuilder::new(context, NativeFunction::from_fn_ptr(get_constant_folding))
-            .name("get constantFolding")
-            .length(0)
-            .build();
-    let set_constant_folding =
-        FunctionObjectBuilder::new(context, NativeFunction::from_fn_ptr(set_constant_folding))
-            .name("set constantFolding")
-            .length(1)
-            .build();
+    let get_constant_folding = FunctionObjectBuilder::new(
+        context.realm(),
+        NativeFunction::from_fn_ptr(get_constant_folding),
+    )
+    .name("get constantFolding")
```

Comment:
Nitpick: can these be indented? It seems inconsistent with the indentation compared to 63 and 68.

Reply:
`rust-fmt` for some reason indents it like that...

Reply:
I had a feeling that might be the case, and I'm guessing writing it similar to 62/63 still corrects because the `FunctionObjectBuilder::new(context.realm(), NativeFunction::from_fn_ptr(set_constant_folding))` is too long.